package.json

```diff
@@ -42,9 +42,9 @@
     "build:lib": "rollup -c",
     "build:ts": "tsc -p tsconfig.build.json",
     "build:copy": "node scripts/build-copy.js",
-    "dev": "concurrently -k -r 'jest --watch' 'yarn run storybook'",
```

Comment:
I'm using `yarn` for the project, so i would prefer if it wasn't changed to `npm`